packages/extension-chakra-storefront/src/setup-app.tsx

```diff
@@ -117,12 +117,12 @@ class ChakraStorefront extends ApplicationExtension<Config> {
             {
                 path: config.pages.ProductList && config.pages.ProductList.path,
                 component: Pages.ProductList
-            },
-            {
-                path: config.pages.Checkout && config.pages.Checkout.path,
-                component: Pages.Checkout,
-                exact: true
             }
+            // {
```

Comment:
Note: temporarily disable cart to reduce noise in the log. Will revert back before merging